web/src/model/topology.ts

```diff
@@ -116,17 +116,17 @@ export const getFilterDefValue = (d: ElementData, t: TFunction) => {
     value = `${d.type}.${d.namespace}.${d.name}`;
   } else if (d.type === 'Node' && (d.host || d.name)) {
     def = findFilter(t, 'host_name')!;
-    value = d.host ? d.host! : d.name!;
+    value = `"${d.host ? d.host! : d.name!}"`;
   } else if (d.type === 'Namespace' && (d.namespace || d.name)) {
     def = findFilter(t, 'namespace')!;
-    value = d.namespace ? d.namespace! : d.name!;
+    value = `"${d.namespace ? d.namespace! : d.name!}"`;
   } else if (d.type && d.name) {
     if (['Service', 'Pod'].includes(d.type)) {
       def = findFilter(t, 'name')!;
     } else {
       def = findFilter(t, 'owner_name')!;
     }
-    value = d.name;
+    value = `"${d.name}"`;
```

Comment:
@jpinsonneau I didn't find how to trigger this filter (when I'm filtering on a pod or service, it uses the "Resource" filter anyway, not that one) , but anyway, it seems legit to have the same exact match being done here too, I guess.

Reply:
On owner scope you will get into this case :wink: